### PR TITLE
Wrap text for long purpose names

### PR DESCRIPTION
**Pull request by anotherminh**
## Related Issues

- Closes https://transcend.height.app/T-28130
Screenshot 2023-08-21 at 12 27 32 PM

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

**Review by kearnie — COMMENTED** [review]
sorry it's been awhile for me in code land but are there safer alternatives to `text-wrap`? it doesnt seem like it's got the best browser coverage https://developer.mozilla.org/en-US/docs/Web/CSS/text-wrap#browser_compatibility

I think we used to do things like `word-wrap` and `white-space` (`overflow-wrap`? `word-break`? lmao theres so many) etc... forget off top of head cc @blake-transcend

**Review by eligrey — DISMISSED** [review]

**Review by michaelfarrell76 — DISMISSED** [review]

File: src/cm.css [hunk 1]
```diff
@@ -170,6 +170,7 @@ a:focus {
   text-shadow: none;
   padding-left: 25px;
   white-space: nowrap;
```
Comment (blake-transcend) [inline src/cm.css:172]: `white-space: nowrap` above (twice nonetheless) will set this to not wrap. We just need to remove those lines and add `word-wrap: break-word`

Comment (anotherminh) [reply src/cm.css:172]: oh nice

**Comment by blake-transcend** [comment]
@anotherminh can you add a screenshot pls

**Comment by anotherminh** [comment]
@blake-transcend updated PR description w screenshot

**Review by blake-transcend — APPROVED** [review]
lgtm

**Pull request merged**